### PR TITLE
Add support to custom labels in operate chart

### PR DESCRIPTION
**Pull request by rafaelcam**

### CODE REVIEW

File: charts/zeebe-operate-helm/templates/service.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "zeebe-operate.fullname" . }}
   labels:
     app: {{ include "zeebe-operate.fullname" . }}
+    {{- toYaml  .Values.labels | nindent 4 }}
```
Comment (Zelldon) [inline charts/zeebe-operate-helm/templates/service.yaml:7]: Is this really what we/you want if you specify a label that the service and the deployment has the same deployment? :thinking: currently not 100% sure.

File: charts/zeebe-operate-helm/templates/deployment.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "zeebe-operate.fullname" . }}
   labels:
     app: {{ include "zeebe-operate.fullname" . }}
+    {{- toYaml  .Values.labels | nindent 4 }}
```
Comment (Zelldon) [inline charts/zeebe-operate-helm/templates/deployment.yaml:7]: TIL about nindent :) but I think newline is not really necessary or?

**Review by Zelldon — COMMENTED** [review]
Thanks @rafaelcam I have smaller questions then we are good to go :)

File: charts/zeebe-operate-helm/templates/deployment.yaml (hunk 2 above)
Comment (rafaelcam) [reply charts/zeebe-operate-helm/templates/deployment.yaml]: Newline fixed. I added a validation for if there is no custom labels.

File: charts/zeebe-operate-helm/templates/service.yaml (hunk 1 above)
Comment (Zelldon) [reply charts/zeebe-operate-helm/templates/service.yaml]: Lets do it when we see a need for it

File: charts/zeebe-operate-helm/templates/deployment.yaml (hunk 2 above)
Comment (Zelldon) [reply charts/zeebe-operate-helm/templates/deployment.yaml]: Not 100% what I meant but is fine i think :)

**Pull request merged**